scripts/build.ts

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env ts-node-transpile-only
+
+import esbuild, { type BuildOptions } from "esbuild";
+import { copyFileSync, readFileSync, writeFileSync, rmSync } from "fs";
+import { globSync as glob } from "glob";
+
+const sharedOptions: BuildOptions = {
+  sourcemap: true,
+  minify: false,
+  allowOverwrite: true,
+  packages: "external",
+};
+
+async function main() {
+  // Start with a clean slate
+  rmSync("pkg", { recursive: true });
```

Comment:
💅🏽 I found this plugin: https://github.com/AkaraChen/esbuild-plugin-clean/blob/main/esm.mjs. 

Maybe we can use it or take it as inspiration for our purposes: `build.onStart(...)`

Reply:
Let's keep this for a future and not add more complexity to this PR